### PR TITLE
Add `io.balena.features.journal-logs` to labels docs

### PR DESCRIPTION
**Pull request by codewithcheese**
Change-type: minor
Signed-off-by: Thomas Manning <thomasm@balena.io>

### CODE REVIEW

File: shared/general/labels.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ io.{{ $names.company.short }}.features.sysfs | false | Bind mounts the host OS `
 io.{{ $names.company.short }}.features.procfs | false | Bind mounts the host OS `/proc` into the container. | v10.8.0 | v2.48.0
 io.{{ $names.company.short }}.features.kernel-modules | false | Bind mounts the host OS `/lib/modules` into the container (i.e. `/lib/modules:/lib/modules`). | v7.23.0 | v2.21.0
 io.{{ $names.company.short }}.features.firmware | false | Bind mounts the host OS `/lib/firmware` into the container. | v7.23.0 | v2.21.0
+io.{{ $names.company.short }}.features.journal-logs | false | Bind mounts journal log directories `/var/log/journal` and `/run/log/journal` aswell as `/etc/machine-id` in read only mode. Required by some logging agents such as `promtail`. Journal logs can be read using libraries such as `sd-journal` in C or `sdjournal` in Go. | vXX.XX | vXX.XX
```
Comment (xginn8) [inline shared/general/labels.md:9]:
```suggestion
io.{{ $names.company.short }}.features.journal-logs | false | Bind mounts journal log directories `/var/log/journal` and `/run/log/journal` as well as `/etc/machine-id` in read only mode. Required by some logging agents such as `promtail`. Journal logs can be read using libraries such as `sd-journal` in C or `sdjournal` in Go. | vXX.XX | vXX.XX
```

Comment (xginn8) [reply shared/general/labels.md:9]: also, should those `xx.xx` be replaced with actual versions?

Comment (codewithcheese) [reply shared/general/labels.md:9]: thanks for the review @xginn8, yep i will update the versions when `meta-balena` has a new version including a new `balena-supervisor`

**Comment by codewithcheese** [comment]
@balena-ci rebase

**Comment by codewithcheese** [comment]
@balena-ci I self-certify!

**Pull request merged**